Infrastructure/gpg-ipdeny/autoscaling_policy.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "instance_min_count": 1,
```

Comment:
I would expect us to have different min_count and max_counts on each environment; is that feasible? how would we go about this?

Reply:
Good point! I updated the script to build the json policy